docs/data.md

```diff
@@ -0,0 +1,51 @@
+# Data Model
+
+## Entity Relational Diagram
+
+```https://mermaid-js.github.io/mermaid-live-editor/
```

Comment:
This doesn't render to md correctly since it's on the first line so it's interpreted as the language for the snippet

Reply:
Unless that's intended, but I'd prefer it to be visible so you know where to paste the code :D